stagecraft/apps/dashboards/views/dashboard.py

```diff
@@ -67,15 +67,9 @@ def single_dashboard_for_spotlight(request, dashboard_slug):
 
 
 def error_response(request, dashboard_slug):
-    error = {
-        'status': 'error',
-        'message': "No dashboard with slug '{}' exists".format(
-            dashboard_slug)
-    }
-    logger.warn(error)
-    error["errors"] = [create_error(request, 404, detail=error['message'])]
-    return HttpResponseNotFound(to_json(error),
-                                content_type='application/json')
+    message = "No dashboard with slug '{}' exists".format(dashboard_slug)
+    logger.setLevel('WARNING')
```

Comment:
Why is the level modified here? `create_http_error` will try to send an `error` message to the logger object.